Week3/moreVehicleToAdvertisementList.js

```diff
@@ -1,17 +1,20 @@
 //11. What if you add one more vehicle to the list, can you have that added to the advertisement without changing the code for question 10?
 
 vehicles = ['motorbike', 'caravan', 'bike', 'truck'];
-console.log(vehicles);
-vehicles.push('car');
-console.log(vehicles);
 
-let advertisedVehicles = '';
+let advertisedVehicles = "Amazing HappyMan's Garage, we service ";
 
-for (let i = 0; i < vehicles.length; i++) {
-  if (i == vehicles.length - 1) {
-    advertisedVehicles += 'and ' + vehicles[i] + 's.';
-  } else {
-    advertisedVehicles += vehicles[i] + 's, ';
+function newAdvertisement(advertisedVehicles, vehicles) {
+  for (let i = 0; i < vehicles.length; i++) {
+    if (i == vehicles.length - 1) {
+      advertisedVehicles += 'and ' + vehicles[i] + 's.';
+    } else {
+      advertisedVehicles += vehicles[i] + 's, ';
+    }
   }
+  return advertisedVehicles;
 }
-console.log(`Amazing HappyMan's Garage, we service ${advertisedVehicles}`);
+
+console.log(vehicles);
+// vehicles.push('car');
```

Comment:
Ah you still have this line commented out, why is that?

(And I think you forgot to copy paste your new function that you made in `advertisement.js` file)

Could you take another look at this? When this is okay then the homework is approved:)!

Reply:
I forgot to modify it :) But I corrected it now.